sdk/digitaltwins/Azure.DigitalTwins.Core/tests/AdtQueryBuilderTests/AdtQueryBuilderTests.cs

```diff
@@ -101,7 +102,7 @@ public void AdtQueryBuilder_Where_Comparison()
                 .Where()
                 .Compare("Temperature", QueryComparisonOperator.GreaterOrEqual, 50)
                 .Build()
-                .Stringify()
+                .GetQueryText()
                 .Should()
                 .Be("SELECT * FROM DigitalTwins WHERE Temperature >= 50");
```

Comment:
Why does this semi column show up as red here? I am curious.

Reply:
not sure. my best guess would be as a result of the weird formatting errors with Build(), but that seems a bit out of place if that's the only semicolon that's different.

Reply:
you can try deleting and re-adding it. see if it shows up differently.